andela/bp-esa-frontend
# #164043698 Migrate to standard create-react-app setup

#### What does this PR do?
Migrate project to use standard create-react-app setup

#### Description of Task to be completed?
- install a new app using create-react-app, including serviceWorker.js
- move babel config from package.json to .babelrc file
- move jest config from package.json to jest.config.js file
- remove redundant webpack modules already provided by create-react-app
- delete package-lock.json and use yarn.lock instead
- delete no-longer needed config files in `/config/` directory
- delete no-longer needed script files in `/scripts/` directory

#### How should this be manually tested?
1. On terminal, run: `yarn test` to observe that all tests pass
2. Launch the application and go to browser to confirm that the UI and functionalities works as before

#### Any background context you want to provide?
This to remove outdated setups previously installed using custom scripts for build and start, which would make it easier for a third party to quickly onboard the codebase and contribute, making it easier to deploy using standard create-react-app deployment

#### What are the relevant pivotal tracker stories?
[#164043698](https://www.pivotaltracker.com/story/show/164043698)
